Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    [TemplatePart(Name = SPLITTERRNAME, Type = typeof(Thumb))]
+    public partial class GridSplitter : Control
```

Comment:
I guess using a template will help other contributors to customise the content of the control in the future, I mean maybe we can give the user the option to add his icons or event controls inside the GridSplitter.